processor/src/test/java/org/mapstruct/ap/testutil/runner/GeneratedSource.java

```diff
@@ -46,32 +46,15 @@
  */
 public class GeneratedSource implements TestRule {
 
+    private final static String FIXTURES_ROOT = "fixtures/";
```

Comment:
I saw the build failure as well 😄. I fixed and I will squash and merge once the build is done